lang/pa_IN.js

```diff
@@ -1,30 +1,36 @@
-//Punjabi language. (India)
+// Punjabi language. (India)
 const strings = {
   nav: {
-    name: "ਕੋਵਿਡ ਹੈਲਪ ਇੰਡੀਆ",
+    name: "ਕੋਵਿਡ ਭਾਰਤ ਦੀ ਮਦਦ ਕਰਦਾ ਹੈ",
     dashboard: "ਡੈਸ਼ਬੋਰਡ",
     feed: "ਫੀਡ",
     signIn: "ਸਾਈਨ ਇਨ ਕਰੋ",
-    signOut: "ਸਾਈਨ ਆਊਟ ਕਰੋ",
+    signOut: "ਸਾਈਨ ਆਊਟ",
+    donation: 'ਦਾਨ',
+    donationLinks: {
+      giveIndia: 'ਇੰਡੀਆ ਆਰਗੇਨਾਈਜ਼ੇਸ਼ਨ ਦਿਓ',
+      ketto: 'ਕੇਟੋ ਸੰਗਠਨ',
+      viratAnushkaFundraiser: 'ਵਿਰਾਟ ਅਤੇ ਅਨੁਸ਼ਕਾ ਫੰਡਰੇਜ਼ਰ'
+    },
   },
```

Comment:
this file overwrote a lot of other strings. Can you have a look at this? @Rohan-Rao

Reply:
yes, sure.